cobalt/h5vcc/h5vcc_updater.idl

```diff
@@ -35,4 +35,13 @@ interface H5vccUpdater {
   // used for testing.
   void setUseCompressedUpdates(boolean use_compressed_updates);
 
+  // Toggles the ability to load self-signed builds in the updater. This should
```

Comment:
It's actually the Crx package (which contains the Cobalt build) that is signed/verified and not the Cobalt build itself, right? It may be good to use that terminology instead (e.g., setAllowSelfSignedPackages).